newscoop/library/Newscoop/Services/IssueService.php

```diff
@@ -94,10 +94,11 @@ public function setIssue(Issue $issue)
      */
     public function issueResolver(Request $request)
     {
-        $uriParts = explode('/', $request->getRequestUri());
-        $uriPartsCount = count(array_filter($uriParts));
+        $uriParts = array_filter(explode('/', $request->getRequestUri()));
+        $uriPartsCount = count($uriParts);
         $issue = null;
-        if ($uriPartsCount >= 2 && $uriPartsCount <= 5) {
+        // Don't try to resolve issues for the admin interface
+        if (!in_array('admin', $uriParts) && $uriPartsCount >= 2 && $uriPartsCount <= 5) {
```

Comment:
why not disabling access to "admin" route in IssueListener ? This will not fire the Issue Listener and will be faster